tests/test_app/library/settings.py

```diff
@@ -200,7 +200,7 @@
     # Related Modal #
     #################
     # Activate Bootstrap modal
-    "related_modal_active": True,
+    "related_modal_active": False,
```

Comment:
Why did you deactivate this for the test app?

Reply:
I've deactivated it everywhere until we solve the mobile issue (modals are not working properly on mobile)

People can still opt to use them if they want, as it's pretty decent on desktop, hopefully we will address the issue in that other PR